utils/roles/remove_node/remove_slurm_node/vars/rocky.yml

```diff
@@ -0,0 +1 @@
+redhat.yml
```

Comment:
rocky files are not created as symbolic link. We need to check all the rocky files are created as symbolic link.